src/Bicep.Cli/Arguments/PublishProviderArguments.cs

```diff
@@ -0,0 +1,97 @@
+        {
+            for (int i = 0; i < args.Length; i++)
+            {
+                var isLast = args.Length == i + 1;
+                switch (args[i].ToLowerInvariant())
+                {
+                    case "--no-restore":
+                        NoRestore = true;
+                        break;
+
+                    case "--target":
+                        if (isLast)
+                        {
+                            throw new CommandLineException("The --target parameter expects an argument.");
+                        }
+
+                        if (this.TargetProviderReference is not null)
+                        {
+                            throw new CommandLineException("The --target parameter cannot be specified twice.");
+                        }
+
+                        TargetProviderReference = args[i + 1];
+                        i++;
+                        break;
+
+                    case "--documentationuri":
+                        if (isLast)
+                        {
+                            throw new CommandLineException("The --documentationUri parameter expects an argument.");
+                        }
+
+                        if (this.DocumentationUri is not null)
+                        {
+                            throw new CommandLineException("The --documentationUri parameter cannot be specified more than once.");
+                        }
+
+                        DocumentationUri = args[i + 1];
+
+                        if (!Uri.IsWellFormedUriString(DocumentationUri, UriKind.Absolute))
+                        {
+                            throw new CommandLineException("The --documentationUri should be a well formed uri string.");
+                        }
+
+                        i++;
+                        break;
+
+                    case "--force":
+                        Force = true;
+                        break;
+
+                    default:
+                        if (args[i].StartsWith("--"))
+                        {
+                            throw new CommandLineException($"Unrecognized parameter \"{args[i]}\"");
+                        }
+
+                        if (IndexFile is not null)
+                        {
+                            throw new CommandLineException($"The input file path cannot be specified multiple times.");
+                        }
+
+                        IndexFile = args[i];
+                        break;
+                }
+            }
+
+            if (IndexFile is null)
+            {
+                throw new CommandLineException($"The input file path was not specified.");
+            }
+
+            if (TargetProviderReference is null)
+            {
+                throw new CommandLineException("The target provider was not specified.");
+            }
+        }
+
+        public string? DocumentationUri { get; }
```

Comment:
Is this used?

Reply:
Wasn't sure if we were planning to use these parameters in the future, but I agree it makes more sense to remove them